main.html

```diff
@@ -29,10 +30,21 @@
         module_frames.push(iframe);
     }
 
-    const bomb = new Bomb(module_frames, 120, 2);
+    const STRIKES_ALLOWED = 2;
+    const BOMB_TIME_S = 2*60;
+
+    const bomb = new Bomb(module_frames, BOMB_TIME_S, STRIKES_ALLOWED);
+
+    const strike_container = document.getElementById("strike_container");
+    for (let i = 0; i < STRIKES_ALLOWED; i++) {
+        strike_container.innerHTML += '<div id="strike">X<div>';
+    }
 
     const timer = document.getElementById("timer");
-    window.addEventListener("tick", () => timer.innerHTML = bomb.time_left_string);
+    
+    window.addEventListener("tick", () => {
+        timer.innerHTML = bomb.time_left_string;
+    });
```

Comment:
why was this expanded again :(

Reply:
om plaats te maken om daar code te zetten in de toekomst ;)